src/components/LeftSideBar.js

```diff
@@ -91,15 +92,24 @@ class LeftSideBar extends Component {
 
   // stores attribute selected
   handleClickAttribute = attribute => {
-    /*   if (this.selectedAttributes !== []) { //this needs to be replaced with add and remove
-            this.setState({
-                selectedAttributes: []
-            });
-            console.log('Clicked!!', attribute, this.state);
-        } else */ {
-      console.log('Clicked!!', attribute, this.state);
-      this.setState({ selectedAttributes: attribute.property_id }); // this.state.selectedAttributes.push(attribute.property_id) });
+    //console.log('====' + this.state.selectedAttributes);
```

Comment:
This event handler gets fired twice, when toggle an attribute, causing it to be briefly added to the array of selected attributes and then immediately removed. Fix suggestion: avoid using `label` tags because the default behavior when clicking on them is that the associated `input` tags also get a click event